apps/XMRig/install-32

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+version=6.16.2
+
+enable_64bit() {
+sudo cat /boot/config.txt &>/dev/null
+if [ "$?" == "1" ]; then
+error 'I was not able to detect /boot/config.txt! Something is very wrong!'
+fi
+echo "arm_64bit=1" | sudo tee --append /boot/config.txt >/dev/null
+echo -e "The 64-bit kernel has been enabled by adding 'arm_64bit=1' to /boot/config.txt\nPlease reboot and install the XMRig app again."
+sleep infinity
+}
+
+config_helper() {
+# Set mining pool
+
+check=0
+until [ "$check" == "1" ]; do
+read -p "What mining pool do you want to use?  Example: gulf.moneroocean.stream:10032 " pool_pref
+read -p "Your mining pool will be set to ${pool_pref}. Is this ok? [Y/n] " pool_conf
```

Comment:
When asking a question, if one of the options is capitalized, (the `Y` in this case), that indicates that it's the default choice if no answer is given. However, in this script, attempting to press Enter for the default choice will trigger errors.

Reply:
Ok, I'll change that. How would one write ENTER in the case statement?

Reply:
> Ok, I'll change that. How would one write ENTER in the case statement?

No, what I'm saying is that I should be able to **press the Enter key** to choose the default choice.

Reply:
So, the case statement would just be something like:
```bash
case $runwiz_conf in
        Y|y|yes|Yes|YES|) check=1; status_green "Starting helper..."; sleep 1; config_helper ;; # Notice the extra `|`
        N|n|no|No|NO) check=1 ;;
        *) check=0; echo 'Invalid input! Please answer y/n, or yes/no.' ;;
esac
```

Reply:
Also fixed in an upcoming commit.

Reply:
With the latest commit, I've changed all `read`-`case` statements so that the user should just be able to hit Enter if there is a recommended answer.